easy_phi/app.py

```diff
@@ -35,6 +36,8 @@
 
 define("default_format", default='json')
 
+#WebSocket object
+global ws
```

Comment:
It is assigned there, and defined here. My point is we don't need `global` keyword